infer/src/base/Config.ml

```diff
@@ -497,6 +498,14 @@ and bootclasspath =
     ~exes:CLOpt.[Toplevel; Java]
     "Specify the Java bootclasspath"
 
+and bo_debug =
+  CLOpt.mk_int ~default:0 ~long:"bo-debug"
+    ~exes:CLOpt.[Toplevel] "Debug mode for buffer-overrun checker"
+
+and bo_filtering =
```

Comment:
Thanks. I change the filtering the filtering by using buckets. 
I think we have addressed all of the issues in this thread. Do you have other comments?